spec/features/api_spec.rb

```diff
@@ -1,5 +1,21 @@
 require 'spec_helper'
 
+shared_examples_for "api request" do
+	context "CORS requests" do
+		it "should set the Access-Control-Allow-Origin header to allow CORS from anywhere" do
+			last_response.headers['Access-Control-Allow-Origin'].should == '*'
```

Comment:
The old `should` syntax has been deprecated. If you look at the test run output, you'll notice that RSpec will complain about this. The new expectation syntax should be used consistently throughout tests.
```ruby
expect(last_response.headers['Access-Control-Allow-Origin']).to eq '*'
```